charts/kafka-lag-exporter/values.yaml

```diff
@@ -88,3 +88,4 @@ prometheus:
   serviceMonitor:
     enabled: false
     interval: 30
+
```

Comment:
Since we don't really have any documentation for the helm chart config, could you add some comments here about the service monitor, along with a commented out sample config including the optional namespace config?

Reply:
* added some comment with link to relvant prometheus helm chart snippet 
* added a commented out sample config

@seglo , I am also planning to add a separate documentation README.md for helm chart like this: https://github.com/helm/charts/tree/master/stable/elasticsearch#configuration
Do you think that will be useful?

Reply:
@abhishekjiitr Yes, that would be super useful.  I've been meaning to do something similar, but it would be wonderful if you could get something started!